deps/+OpenEXR/OpenEXR.patch

```diff
@@ -28,3 +28,15 @@
  #include <ImfMisc.h>
  #include <ImfHeader.h>
  #include <ImfAttribute.h>
+
+--- ../OpenEXR-orig/OpenEXR/IlmImf/ImfSimd.h	2021-02-12 17:56:19.000000000 +0100
++++ ./OpenEXR/IlmImf/ImfSimd.h	2023-06-01 13:22:15.777480000 +0200
+@@ -45,7 +45,7 @@
+ 
+ 
+ // GCC and Visual Studio SSE2 compiler flags
+-#if defined __SSE2__ || (_MSC_VER >= 1300 && !_M_CEE_PURE)
++#if defined __SSE2__ || (_MSC_VER >= 1300 && !_M_CEE_PURE) && !defined(_M_ARM64)
```

Comment:
This is hopefully fixed in OpenEXR 3.2.0.

Reply:
Apparently so, the file is now way more comprehensive and even has NEON optimizations https://github.com/AcademySoftwareFoundation/openexr/blob/v3.3.0/src/lib/OpenEXR/ImfSimd.h